scripts/qp_import_trexio.py

```diff
@@ -195,7 +195,7 @@ def write_ezfio(trexio_filename, filename):
             prim_factor  = trexio.read_basis_prim_factor(trexio_file)
             for i,p in enumerate(prim_factor):
                 coefficient[i] *= prim_factor[i]
-            ezfio.set_ao_basis_primitives_normalized(True)
+            ezfio.set_ao_basis_primitives_normalized(False)
```

Comment:
@scemama @Ydrnan Should not this depend on the contents of the TREXIO file? What if the file was produced by the code that does normalize AO basis primitives before writing them (which was the case I think for the cp2k)?

Reply:
I think that Yann is correct. In QP, `primitives_normalized` means that the coefficients of the AOs suppose that the primitives are normalized. In TREXIO, we store the normalization explicitly, so we don't assume that they are normalized. Now, I need to check again that everything is correct.

Reply:
I think that the problem comes from what QP writes in the TREXIO file.
`trexio check-basis` is not always correct, so there is a bug in what is written.

Reply:
I see. Indeed, if you eventually read the normalization factors from TREXIO and handle them accordingly - then there is no problem.

Reply:
I just made a new PR that fixes all the problems